oml/interfaces/miners.py

```diff
@@ -22,34 +22,35 @@ def labels2list(labels: TLabels) -> List[int]:
 
 
 class ITripletsMiner(ABC):
-    """An abstraction of inbatch triplet miner."""
+    """
+    An abstraction of triplet miner.
```

Comment:
Seems like it is important to clarify that mining taking place in batch.

Reply:
It's not true since we have Miner with memory
That is why I updated the doc